### ISSUE TITLE
Task Manager is unstable

### ISSUE THREAD

**Issue by jgritten** [AveYo/MediaCreationTool.bat#278]
Since performing the update, Task Manager has been unreliable at best. 

- Performance tab doesn't update
- Details list wont populate
- Becoming unresponsive(Not Responding) at random points
- Received an email in regard to a windows defender warnings where a threat has been detected yet when I open defender, no issues are found

Things I have done to troubleshoot include but are not limited to

- SFC and DISM scans
- Check for and install all updates
- Registry Task Manager reset
- Defender Scans including, quick, full and offline

Any help or thoughts on things I could try would be great. Thanks for your time

**Comment by AveYo** [comment]
The version you got would be nice to know.  

The issue might be the Smart App Control feature that you should disable in Defender - App & browser control settings. Also turn off Defender - Device security - Core isolation - Memory Integrity.

**Comment by jgritten** [comment]
Smart App Control is not listed under the App & Browser control within Windows Security, and Core Isolation gives me a Page not available: the page you are trying to access has no supported features and is not available.

And for the version question, which version are you after? The version of your utility I used, the version of windows that I'm running or something else? I only downloaded the utility from here the same day I commented, so whichever version is prob currently available.
